vvplot error

Thanks for the code. I just met a problem.
I am running it on wsl ubuntu 20.04. It went nornally at the very beginning, but when I was trying to vvplot the results, no visualized results occurred followed by such error:

vvplot results_re600_n350/000400.h5 bvs.png -x -2,18 -BVS
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error running gnuplot: No such file or directory
make: *** [Makefile:17: bvs.png] Aborted
make: *** Deleting file 'bvs.png'

I have no idea how to fix it. Do you have any suggestions? (limited memory? etc.) Thanks~

--- a/README.md
+++ b/README.md
@@ -219,11 +219,11 @@ vvxtract stepdata_re600_n350.h5 time body00/force_hydro \
 
 ## Building from source
 
-Install build dependencies:
+Install build and runtime dependencies:
 
 ```bash
 sudo apt-get update
-sudo apt-get install build-essential cmake make git liblapack-dev
+sudo apt-get install build-essential cmake make git liblapack-dev gnuplot
 ```
 
 Build the project:

